my-react-app/src/translation/en_translation.json

```diff
@@ -0,0 +1,13 @@
+{
+    "submit": "submit",
```

Comment:
die übersetzungen in english können gerne groß geschrieben werden also zb Phone Number